control/src/uart_Kangaroo_interface.c

```diff
@@ -0,0 +1,76 @@
+#include "uart.h"
+
+#include <stdio.h>
+#include <unistd.h>
+#include <string.h>
+
+void kangarooInterfaceSetup()
+{
+    unsigned char temp1[] = "1,start";
+    unsigned char temp2[] = "2,start";
+    unsigned char temp3[] = "1,units 100 cm = 2048 lines";
+    unsigned char temp4[] = "2,units 100 cm = 2048 lines";
```

Comment:
temp1-4 are not appropriate names for variables. This can be done with #define constants or with a single character array that you copy a new string into each time

Reply:
there more so temporary names than "temp" variables

Reply:
Changed this so that it is hard coded. I will probably not be able to change that before the demo, however, I will write comments and say some things during the demo about how it should be done